specification/langRef/attributes/universalAttributes.dita

```diff
@@ -128,8 +128,9 @@
         </dlentry>
         <dlentry id="id">
           <dt id="attr-id"><xmlatt>id</xmlatt></dt>
-          <dd>An anchor point. This ID is the target for references by <xmlatt>href</xmlatt> and
-              <xmlatt>conref</xmlatt> attributes and for external applications that refer to DITA
+          <dd id="reusable-id">An anchor point. This ID is the target for references by <ph platform="dita"><xmlatt>href</xmlatt> and
+            <xmlatt>conref</xmlatt></ph> <ph platform="lwdita"><b>href</b> and
+              <b>conref</b></ph> attributes and for external applications that refer to DITA <ph platform="lwdita">or LwDITA</ph>
```

Comment:
`<b>href</b>` should be `<xmlatt>href</xmlatt>`.

`<b>conref</b>` should be `<xmlatt>conref</xmlatt>`